translations/messages.bg.yaml

```diff
@@ -167,23 +167,23 @@ add_new_video: Добавяне на ново видео
 rules: Правила
 columns: Колони
 alphabetically: По азбучен ред
-up_votes: Подсилвания
+up_votes: Препубликации
```

Comment:
You sure about this? It's actually more Подсилвания

Reply:
I'm not sure. I copied `Препубликуване` (Boost) from the Bulgarian translation of Twitter, because `Подсилване` sounds unnatural to me. `Препубликации` (Boosts) is wrong, I should have translated it as `Препубликувания`. I will change them to `Подсилване` and `Подсилвания` in Weblate. Mastodon also uses `Подсилвания`.

Reply:
Thanks!